Resources/Prototypes/Entities/Structures/Furniture/rollerbeds.yml

```diff
@@ -52,9 +52,12 @@
       buckleOffset: "0,0.15"
       unbuckleOffset: "0,0.15"
     - type: Appearance
+    - type: GenericVisualizer
```

Comment:
doesnt this override the foldable gen vis? would have to copy the enum here (you could also use visibility so you only have to copypaste here and map the _buckled to an invisible layer in the inheritors)